src/main/java/uk/gov/justice/hmpps/prison/service/support/LocationProcessor.java

```diff
@@ -104,34 +104,38 @@ public static Location processLocation(final Location location, final boolean pr
         final String newDescription = newDescription(location, preferUserDescription);
 
         return Location.builder()
-                .agencyId(location.getAgencyId())
-                .currentOccupancy(location.getCurrentOccupancy())
-                .description(formatDescription ? formatLocation(newDescription) : newDescription)
-                .locationId(location.getLocationId())
-                .locationPrefix(newLocationPrefix)
-                .locationType(location.getLocationType())
-                .operationalCapacity(location.getOperationalCapacity())
-                .parentLocationId(location.getParentLocationId())
-                .userDescription(formatLocation(location.getUserDescription()))
-                .internalLocationCode(location.getInternalLocationCode())
-                .subLocations(location.getSubLocations())
-                .build();
+            .agencyId(location.getAgencyId())
+            .currentOccupancy(location.getCurrentOccupancy())
+            .description(formatDescription ? formatLocation(newDescription) : newDescription)
+            .locationId(location.getLocationId())
+            .locationPrefix(newLocationPrefix)
+            .locationType(location.getLocationType())
+            .operationalCapacity(location.getOperationalCapacity())
+            .parentLocationId(location.getParentLocationId())
+            .userDescription(formatLocation(location.getUserDescription()))
+            .internalLocationCode(location.getInternalLocationCode())
+            .subLocations(location.getSubLocations())
+            .build();
     }
+
     public static List<LocationSummary> processLocationSummaries(final List<LocationSummary> locations) {
         return locations.stream().map(loc -> processLocationSummary(loc)).toList();
     }
 
     public static LocationSummary processLocationSummary(final LocationSummary locationSummary) {
         return LocationSummary.builder()
             .locationId(locationSummary.getLocationId())
-            .userDescription(
-                StringUtils.defaultIfBlank(locationSummary.getUserDescription(),
-                    stripAgencyId(locationSummary.getDescription(), locationSummary.getAgencyId())))
+            .userDescription(formatUserDescription(locationSummary))
             .description(locationSummary.getDescription())
             .agencyId(locationSummary.getAgencyId())
             .build();
     }
 
+    private static String formatUserDescription(LocationSummary locationSummary) {
+        return StringUtils.defaultIfBlank(formatLocation(locationSummary.getUserDescription()),
+            stripAgencyId(locationSummary.getDescription(), locationSummary.getAgencyId()));
```

Comment:
is the user description always set?  Wondering if the `formatLocation` needs to go around the `getDescription` too

Reply:
Well I wondered that but the Description is always stuff like CSI-A-1-001 which doesn't make sense to make lower case

Reply:
fyi asked Eddie, he he happy with the normal description staying as it is (i.e. just the user description is formatted)